src/raiden_libs/logging.py

```diff
@@ -34,14 +35,22 @@ def setup_logging(log_level: str) -> None:
     )
 
 
-def log_event(event: Event) -> Dict[str, Any]:
-    event_data = asdict(event)
+def change_bytes(event_dict: Dict, key: Any, val: bytes) -> Dict[str, Any]:
+    if len(val) == 20:
+        event_dict[key] = to_checksum_address(val)
+    else:
+        event_dict[key] = to_hex(val)
+    return event_dict
 
-    for key, val in event_data.items():
-        if isinstance(val, bytes):
-            if len(val) == 20:
-                event_data[key] = to_checksum_address(val)
-            else:
-                event_data[key] = to_hex(val)
 
-    return event_data
+def format_to_hex(_logger: Any, _log_method: Any, event_dict: Dict) -> Dict[str, Any]:
+    for key, val in event_dict.items():
+        if isinstance(val, bytes):
+            change_bytes(event_dict, key, val)
+        if isinstance(val, Event):
```

Comment:
In this case it would be nice to output the event name as well

Reply:
Wouldn't that be another logging processor? This should only change bytes to hex and neither delete or add any info, right?

Reply:
but you're changing it here: `event_data = asdict(val)`

Reply:
So when I understand you correctly, then you suggest to add another structlog processor that also includes the Event name. Or would you inlcude that in the `format_to_hex` function?

Reply:
I'd include it there, as you loose the name when you convert it to a dictionary.

Reply:
finally successfully implemented